src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssuePage.kt

```diff
@@ -82,4 +82,12 @@ class IssuePage(
 
     fun getIssueId(): Long = driver.findElement(By.id("key-val")).getAttribute("rel").toLong()
     fun getIssueType(): String = driver.findElement(By.id("type-val")).text
+
+    internal fun openCommentTabPanel(): CommentTabPanel {
+        return CommentTabPanel(driver).waitForActive()
+    }
+
+    internal fun openHistoryTabPanel(): HistoryTabPanel {
```

Comment:
- [ ] Consider removing `internal`

Reply:
This came up when we were reviewing japicmp report on a zoom call yesterday. The `internal` part got removed, so I'm closing the conversation and marking the task.